lib/widgets/selection.dart

```diff
@@ -1627,7 +1627,12 @@ class _GoToArtistSelectionAction extends StatelessWidget {
     if (content is Song) {
       HomeRouter.instance.goto(HomeRoutes.factory.content(content.getArtist()));
     } else if (content is Album) {
-      HomeRouter.instance.goto(HomeRoutes.factory.content(content.getArtist()));
+      final artist = content.getArtist();
+      if (artist != null) {
+        HomeRouter.instance.goto(HomeRoutes.factory.content(artist));
+      } else {
+        ShowFunctions.instance.showToast(msg: getl10n(controller.context).artistUnknown);
```

Comment:
Why do we even show this button in the first place then?
And can't we heuristically choose the artist from the first song on album?